build/pipelines/release.yml

```diff
@@ -163,6 +174,10 @@ jobs:
       arguments: -MarkdownNoticePath .\NOTICE.md -OutputPath .\src\cascadia\CascadiaPackage\NOTICE.html
       pwsh: true
   - ${{ if eq(parameters.buildTerminal, true) }}:
+    - pwsh: |-
+        ./build/scripts/Patch-ManifestsToWindowsVersion.ps1 -NewWindowsVersion "10.0.22000.0"
+      displayName: Update manifest target version to Win11 (if necessary)
+      condition: and(succeeded(), eq(variables['TerminalTargetWindowsVersion'], 'Win11'))

```

Comment:
Since most of us are working on Windows 11 right now, wouldn't it be more helpful to default to 22000 and instead set a lower version for the Windows 10 build?

Reply:
I'd like to let folks who are still running Windows 10 press F5 and deploy our project -- the "default" build mode should be Win10 for this reason :)